taxi/views.py

```diff
@@ -87,3 +92,47 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, CreateView):
+    model = get_user_model()
+    fields = [
+        "username",
+        "first_name",
+        "last_name",
+        "license_number",
+        "password"
+    ]
+    success_url = reverse_lazy("taxi:driver-list")
+
+    def form_valid(self, form):
+        user = form.save(commit=False)
+        user.set_password(form.cleaned_data["password"])
+        user.save()
+        return super().form_valid(form)
```

Comment:
The `form_valid` method in `DriverCreateView` correctly uses `set_password()` to hash the password before saving the user, which is a crucial security measure. This addresses the previous concern about password hashing.